### ISSUE TITLE
Faulty JSON when getting a list of users through CLI tool

### ISSUE THREAD

**Issue by esbenboye** [Azure/azure-xplat-cli#2226]
I'm pulling out a list of users, using the CLI command  `azure ad user list --json`(using the CLI Tool for Mac), but the json I get back is faulty. 
At line 799 (After exactly 100 objects - Don't know if this means anything), I get an error, where an array is ended, and a new one is started:
(Real values replaced by me, and I've cut the result down to exactly where the problem is)

```
  {
      "objectId": "...",
      "objectType": "User",
      "userPrincipalName": "...",
      "displayName": "...",
      "signInName": "...",
      "mail": "..."
    }
  ]
  [
    {
      "objectId": "...",
      "objectType": "User",
      "userPrincipalName": "...",
      "displayName": "...",
      "signInName": "...",
      "mail": "..."
    }
```

This poses quite a problem since it pretty much makes the response useless, since it can't be parsed into  an object/array.

**Comment by yugangw-msft** [comment]
@vivsriaus , please investigate

**Comment by amarzavery** [comment]
Fixed in https://github.com/Azure/azure-xplat-cli/pull/2944

**Comment by ravbhatnagar** [comment]
@amarzavery - This is not owned by the ARM team. Please assign it to AD owner

**Comment by amarzavery** [comment]
@ravbhatnagar - Thanks for notifying.
@esbenboye This issue should be resolved in the latest version 0.10.7 of xplat cli. Please let us know if you find any issues.
